setup.cfg

```
@@ -120,6 +120,10 @@ windows_platform =
 osx_platform =
     %(macos_platform)s
 
+posix_platform =
```

Is this posix or GTK?

I know that we have `macos_platform` but that is because macOS is an OS in which you are force fed :)

In linux, you are free to use a posix system without gtk :) and many servers might not have or need GTK

```suggestion
gtk_platform =
```

---

Yeah that makes sense. I updated this and relevant lines.